lib/cisco_node_utils/vxlan_vtep_vni.rb

```diff
@@ -24,13 +24,19 @@
 module Cisco
   # VxlanVtepVni - node utility for vxlan vtep vni members.
   class VxlanVtepVni < NodeUtil
-    attr_reader :name, :vni
+    attr_reader :name, :vni, :vrf
 
-    def initialize(name, vni, instantiate=true)
+    def initialize(name, vni, vrf=false, instantiate=true)
```

Comment:
nit: I would use `assoc_vrf` here, only because vrf means something completely different in all of our other providers. Your call.